bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java

```diff
@@ -585,7 +586,10 @@ private void parseBehavior(Node node, Pin outpin, String lines, DataFlowDiagram
         var behavior = node.getBehaviour();
         for (String behaviorString : behaviorStrings) {
             if (behaviorString.contains("forward ")) {
-            	 List<String> packets = Arrays.asList(behaviorString.split(" ")[1].split(DELIMITER));
+            	 var assignment = ddFactory.createForwardingAssignment();
+            	 assignment.setOutputPin(outpin);
+            	 
+            	 List<String> pinNames = Arrays.asList(behaviorString.split(" ")[1].split(DELIMITER_MULTI_PIN));
```

Comment:
Using `,` is wrong, you can only forward one pin in the web dfd. hence why i use the same delimiter from everywhere else(`|`)

Reply:
"Forward data, encrypt" is a valid behavior if data and encrypt are separate pins with input flows.

That's the case right now and what we described in #193.

It's also valid to use:
Forward data
Forward encrypt

Reply:
Would `forward auth_server|login_server,db` work correctly with this code?

Reply:
Yes

Reply:
If you convert it back and forth, though, it would turn into two separate statements